hotel/models/hotel.py

```diff
@@ -19,16 +19,15 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>
 #
 # ---------------------------------------------------------------------------
-from openerp.exceptions import except_orm, UserError, ValidationError
-from openerp.tools import misc, DEFAULT_SERVER_DATETIME_FORMAT
-from openerp import models, fields, api, _
-from openerp import workflow
+from odoo.exceptions import except_orm, UserError, ValidationError
+from odoo.tools import misc, DEFAULT_SERVER_DATETIME_FORMAT
+from odoo import models, fields, api, _
+from odoo import workflow
 from decimal import Decimal
 import datetime
 import urllib2
 import time
```

Comment:
The import order is wrong.
See https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0